splink/comparison_level_library.py

```diff
@@ -15,6 +20,8 @@ def __init__(self, col_name, valid_string_regex: str = None) -> ComparisonLevel:
             col_name (str): Input column name
             valid_string_regex (str): regular expression pattern that if not
                 matched will result in column being treated as a null.
+            invalid_dates_as_null (bool): If True, set all invalid dates to null.
+                Defaults to false.
```

Comment:
Maybe here and the other argument there could be a bit of elaboration on how `valid_string_regex` format changes depending on this flag is set

Reply:
Yea makes sense, can flesh it out